packages/twenty-server/src/core/open-api/open-api.service.ts

```diff
@@ -62,4 +62,11 @@ export class OpenApiService {
 
     return schema;
   }
+
+  async generateMetaDataSchema(): Promise<OpenAPIV3.Document> {
+    //TODO Rest MetaData api?
```

Comment:
The REST endpoint do not exit yet no :-(

https://github.com/twentyhq/twenty/issues/3648